client/client_update.go

```diff
@@ -160,8 +170,20 @@ func processUpdateResponse(response *http.Response) (interface{}, error) {
 	}
 }
 
-func makeUpdateCheckRequest(server string) (*http.Request, error) {
-	url := buildApiURL(server, "/deployments/device/deployments/next")
+func makeUpdateCheckRequest(server string, current CurrentUpdate) (*http.Request, error) {
+	vals := url.Values{}
+	if current.DeviceType != "" {
+		vals.Add("device_type", current.DeviceType)
+	}
+	if current.Artifact != "" {
+		vals.Add("artifact", current.Artifact)
```

Comment:
No, it's `artifact` according to the spec https://tracker.mender.io/browse/MEN-744